balancer/balancer.go

```diff
@@ -327,6 +333,13 @@ type Picker interface {
 // UpdateClientConnState, ResolverError, UpdateSubConnState, and Close are
 // guaranteed to be called synchronously from the same goroutine.  There's no
 // guarantee on picker.Pick, it may be called anytime.
+//
+// NOTICE: This interface is intended to be implemented by gRPC, or intercepted
```

Comment:
This was supposed to be on `ClientConn`, not `Balancer`, right?

Reply:
Yes, I pasted it at the wrong place. Removed from this PR, fixing it in https://github.com/grpc/grpc-go/pull/8026.